src/com/session/Check.java

```diff
@@ -47,13 +52,23 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
 			} else {
 				count = 0;
 
-			}
+			}*/
+		
+		
+		Filter checkEmail	= new FilterPredicate("Email", FilterOperator.EQUAL, email);
```

Comment:
this whole check can be put in a method, also make that method return type as Entity, so if auth pass it should return fetched entity, if not return null